src/main/game/KeyEventController.java

```diff
@@ -18,10 +18,12 @@
     Game game;
```

Comment:
I didn't it was there before, and the cycling functions in the game class are protected.